management/guardian_test.go

```diff
@@ -262,6 +263,49 @@ func TestGuardian(t *testing.T) {
 				assert.Equal(t, expectedCustomApp.GetAppleAppLink(), actualCustomApp.GetAppleAppLink())
 				assert.Equal(t, expectedCustomApp.GetGoogleAppLink(), actualCustomApp.GetGoogleAppLink())
 			})
+
+			t.Run("DirectAPNS", func(t *testing.T) {
+				configureHTTPTestRecordings(t)
+
+				_, err := api.Guardian.MultiFactor.Push.DirectAPNS()
+				assert.NoError(t, err)
+
+				// Cannot reflect back the payload we initially received
+				// as it is incompatible
+
+				// Read the p12 from file
+				expectedP12, err := ioutil.ReadFile("../test/data/apns.p12")
```

Comment:
I opted to use io/ioutil, in keeping with the style of the current project and not introducing any new dependencies, as you can see, it's used by https://github.com/auth0/go-auth0/blob/main/management/gen-methods.go#L22 , happy to change out to os.ReadFile if you wish

Reply:
Yeah it will need to change if you can please in order to pass the linting. The gen-methods.go is ignored by the linter it seems, but I'll look to get that updated to keep it consistent.